external-import/comlaude/src/main.py

```diff
@@ -10,7 +10,8 @@
 
 import yaml
 from pycti import OpenCTIConnectorHelper, get_config_variable
-from stix2 import Bundle, DomainName, Indicator, Relationship
+from stix2 import Bundle, DomainName, Indicator, Relationship, Identity
+from stix2.v21 import TLP_AMBER
```

Comment:
You don't need to specify the stix2.v21 import for TLPs:
Example :
either “stix2.TLP_AMBER” is enough
either in the import "from stix2 import TLP_AMBER"

Reply:
Hello Megrafredo, ok it's noted, i will modified the script